lib/macro/compileTypeSuite.ts

```diff
@@ -0,0 +1,24 @@
+import { Compiler, ICompilerOptions } from "../index";
+import { macroError, macroInternalError } from "./errors";
+
+export type ICompilerArgs = [string, ICompilerOptions];
+
+export function compileTypeSuite(args: ICompilerArgs): string {
+  let compiled: string | undefined;
+  const [file, options] = args;
+  const optionsString = JSON.stringify(options);
+  const context = `compiling file ${file} with options ${optionsString}`;
+  try {
+    compiled = Compiler.compile(file, options);
+  } catch (error) {
+    throw macroError(`Error ${context}: ${error.name}: ${error.message}`);
+  }
+  const exportStatement = compiled.split("\n").slice(2).join("\n");
```

Comment:
A comment would help that we are processing js:cjs format here, which is expected to start with such and such lines.

Reply:
Added some explanatory comments here in f8bd4a16e7d764978f733845a59fc9c70014d7f4